test/spec/features/app/ps_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+describe 'app ps' do
```

Comment:
There's no guarantee that there is anything deployed, does it make sense to check the `app ps` without anything deployed? (IMO no)